src/Native/Bootstrap/main.cpp

```diff
@@ -372,6 +372,11 @@ extern "C" void RhpBulkWriteBarrier()
 {
     throw 42;
 }
+
+extern "C" Object* RhNewArrayAsString(MethodTable*)
```

Comment:
RhNewArrayAsString helper should be exported under the RhNewArray name:

        [RuntimeImport(RuntimeLibrary, "RhNewArray")]
        internal static extern String RhNewArrayAsString(EETypePtr pEEType, int length);